### DISCUSSION TITLE
Has anyone tried using LMQL with Azure OpenAI ?

### DISCUSSION THREAD

**Discussion by younes-io** [eth-sri/lmql#306] [Q&A] [answered]
As per [the docs](https://lmql.ai/docs/models/azure.html#configuration-via-lmql-model), I tried to use LMQL with my Azure OpenAI instance, and it fails: https://github.com/eth-sri/lmql/issues/305

Has anyone tried the configuration via lmql model ?

Thank you,

**Comment by lbeurerkellner** [comment, marked as answer]
Resolved as per #305.
